src/NServiceBus.Unity/FodyWeavers.xml

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Weavers>
-  <Janitor />
+<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">
```

Comment:
You can instead add `GenerateXsd="false"` here and prevent Fody from generating the FodyWeavers.xsd file below.